pconvert_rust/test/blending.js

```diff
@@ -0,0 +1,86 @@
+const fs = require("fs");
+const path = require("path");
+const assert = require("assert");
+const pconvert = require("../../pkg/pconvert_rust.js");
+
+const TEST_ASSETS = path.resolve("assets/test");
+
+describe("NodeJS WASM", async function() {
+    it("should have known module constants", () => {

```

Comment:
Replicate the test structure we've used in ripe-twitch (and in all other JS projects):
- General describe
- Describe `#someFunction`
- `it`s

Reply:
Also, are the tests the same as https://github.com/ripe-tech/pconvert-rust/blob/b7870bf89f5fe06f653ba5c6d05c3cd39f08af32/pconvert_rust/test/blending.py ? They probably should be a copy-cat of oneanother.

Reply:
More or less. Key differences:
- no multi-threading in the nodeJS version
- module constants only exist in the nodeJS version

Reply:
Can't you use multi-threading here?

Why are the constants exclusive to this version?

Reply:
Actually, I think the python module has constants, I can test that. There just isn't a `getModuleConstants`.
The multi-threading write crate does not compile to WASM. Because the WASM was more of a gimmick, not multi-thread was implemented for the module. In fact, most macros to decide which files to compile based on the crate activated features cut out the multi-threaded code for a WASM target.

Reply:
They should be closer to each other tho. Some tests are missing in the python branch. This one is more complete. I am going to mimic them as best as possible.